goleak is no longer compatible with Go 1.5+

[Readme](https://github.com/uber-go/goleak#installation) explains this is compatible with Go `1.5+`.
However, the latest version `v1.2.0` only works on Go `1.18+`.

Sample code:

```go
package main

import (
	"testing"

	"go.uber.org/goleak"
)

func TestLeak(t *testing.T) {
	defer goleak.VerifyNone(t)
}
```

```bash
$ go version
go version go1.13.15 linux/amd64
$ go test
# go.uber.org/goleak
/go/pkg/mod/go.uber.org/goleak@v1.2.0/options.go:115:3: undefined: isTraceStack
note: module requires Go 1.18
FAIL    example.com/foo [build failed]
```

The previous version `v1.1.12` works expectedly. This seems due to https://github.com/uber-go/goleak/pull/79.
Is current version only compatible with Go 1.18+?

--- a/README.md
+++ b/README.md
@@ -8,7 +8,9 @@ You can use `go get` to get the latest version:
 
 `go get -u go.uber.org/goleak`
 
-`goleak` also supports semver releases. It is compatible with Go 1.5+.
+`goleak` also supports semver releases.
+
+Note that go-leak only [supports][release] the two most recent minor versions of Go.
 
 ## Quick Start
 
@@ -69,3 +71,4 @@ No breaking changes will be made to exported APIs before 2.0.
 [ci]: https://github.com/uber-go/goleak/actions/workflows/go.yml
 [cov-img]: https://codecov.io/gh/uber-go/goleak/branch/master/graph/badge.svg
 [cov]: https://codecov.io/gh/uber-go/goleak
+[release]: https://go.dev/doc/devel/release#policy

